plugins/window_size/windows/window_size_plugin.cpp

```diff
@@ -64,9 +64,10 @@ EncodableValue GetPlatformChannelRepresentationForMonitor(HMONITOR monitor) {
   MONITORINFO info;
   info.cbSize = sizeof(MONITORINFO);
   GetMonitorInfo(monitor, &info);
-  UINT dpi_x, dpi_y;
-  GetDpiForMonitor(monitor, MDT_EFFECTIVE_DPI, &dpi_x, &dpi_y);
-  double scale_factor = dpi_x / kBaseDpi;
+  // Send a nullptr since the top-level window hasn't been created. This will
+  // get the neares monitor's DPI.
```

Comment:
primary

But this isn't the right thing to call; we want the DPI for `monitor`, which is why I had you expose the other function too.